R/fwrite.R

```diff
@@ -0,0 +1,62 @@
+fwrite <- function(dt, file.path, append = FALSE, quote = TRUE,
```

Comment:
1) ok, I'll change this

2) This is semi-intentional. Supporting `file=""` (stdout) and connections would require passing file handles (instead of just file name strings) from R to C and I don't currently know how to do this. R's C interface is poorly documented.